data/human/human missions.txt

```diff
@@ -7453,3 +7453,233 @@ mission "Adelita 5"
 			`Ale, River, and Bunker look glad to be back on <planet>. "Being there in person was bad enough," Ale grumbles as she looks through a stack of folders on her datapad. "Now we have to watch it over and over again to edit it into something presentable."`
 			`	As they are leaving, Ale hands you a stack of credit chips totaling <payment>. "There's a bonus in there for you. We really couldn't have done it without you," she tells you with a bittersweet smile. "Of course, you'll be one of the first to know when it's done."`
 			`	She starts off to join River and Bunker out in the spaceport before turning around one last time in the doorway. "We might be in need of you for some future projects too, Captain <last>. But goodbye for now. Stay safe on the star lanes!"`
+
+mission "Syndicate Business 5.1 A 10m Expansion Update"
+	landing
+	to offer
+		has "Syndicate Business 5: done"
+		has "event: 10m investment done"
+		or
+			has "Syndicate Business expansion 10m good"
+			has "Syndicate Business expansion 10m neutral"
+			has "Syndicate Business expansion 10m evil"
```

Comment:
```suggestion
			has "Syndicate Business expansion 10m evil"
		random < 50
```

Reply:
Hmmm, having the random aspect here means that this conversation can flag _after_ the other human focused one, which isn't ideal as that references this first one ("noteworthy incident", etc). Would it be okay to keep this at 100% and only add a random aspect to the 2nd conversation?

Reply:
Yes, makes sense.